pkg/network/http/batch_manager.go

```diff
@@ -129,5 +129,9 @@ func (m *batchManager) GetPendingTransactions() []httpTX {
 		}
 	}
 
+	for _, tx := range transactions {
+		tx.PostProcess()
```

Comment:
Could we do this in `httpBatch.Transactions()`? That way you cannot forget to call this function and have incomplete/bad data.

Reply:
I believe that is the only function creates `httpTx` objects, but if there are others we should do the same there too.